marlowe-dashboard-client/README.md

```diff
@@ -1,83 +1,105 @@
+
+* [Cardano node](https://github.com/input-output-hk/cardano-node)
+* [marlowe-pab](https://github.com/input-output-hk/marlowe-cardano/tree/master/marlowe/pab)
+* [dashboard-server](https://github.com/input-output-hk/marlowe-cardano/tree/main/marlowe-dashboard-server)
+* chain-index
+* [wallet backend](https://github.com/input-output-hk/cardano-wallet)
 
-Marlowe Run is written in [PureScript](https://www.purescript.org/) and uses npm and [spago](https://github.com/purescript/spago) for managing dependencies. It talks to the [marlowe-pab](https://github.com/input-output-hk/marlowe-cardano/tree/master/marlowe/pab) which also needs to be up and running during development. The client build also depends on PureScript files that are generated by the backend service using [purescript-bridge](https://github.com/eskimor/purescript-bridge).
+The [marlowe-pab](https://github.com/input-output-hk/marlowe-cardano/tree/master/marlowe/pab) is a version of the [plutus-pab](https://github.com/input-output-hk/plutus-apps/tree/master/plutus-pab) with the Marlowe Plutus contracts built-in.
 
-The [marlowe-pab](https://github.com/input-output-hk/marlowe-cardano/tree/master/marlowe/pab) is a version of the [plutus-pab](https://github.com/input-output-hk/plutus-apps/tree/master/plutus-pab) with the Marlowe Plutus contracts built in.
+**Note 1**: _The current version of the application is an alpha release and it is targeted for developers to provide the necesary feedback to shape the final product. We encourage developers to try this on a testnet and [submit issues](https://github.com/input-output-hk/marlowe-cardano/issues/new/choose) to this repository._
 
-**Note**: _The workflow described here relies heavily on Nix. This means you should either be working inside a nix-shell environment or use tools such as [lorri](https://github.com/target/lorri) or [nix-direnv](https://github.com/nix-community/nix-direnv) or similar to provide a suitable environment._ If you are using Mac and you run into a segmentation fault running nix-shell, you can try running it as `GC_DONT_GC=1 nix-shell` as this [issue suggest](https://github.com/NixOS/nix/issues/4246).
+**Note 2**: _The workflow described here relies heavily on Nix. This means you should either be working inside a nix-shell environment or use tools such as [lorri](https://github.com/target/lorri) or [nix-direnv](https://github.com/nix-community/nix-direnv) or similar to provide a suitable environment. Make sure you have configured the [nix binary cache](https://github.com/input-output-hk/marlowe-cardano#how-to-set-up-the-iohk-binary-caches) to avoid unnecesary builds. If you are using Mac and you run into a segmentation fault running nix-shell, you can try running it as `GC_DONT_GC=1 nix-shell` as this [issue suggest](https://github.com/NixOS/nix/issues/4246)._
```

Comment:
I would embolden this point, as people seem to miss it 
```suggestion
**Note 2**: _The workflow described here relies heavily on Nix. This means you should either be working inside a nix-shell environment or use tools such as [lorri](https://github.com/target/lorri) or [nix-direnv](https://github.com/nix-community/nix-direnv) or similar to provide a suitable environment. **Make sure you have configured the [nix binary cache](https://github.com/input-output-hk/marlowe-cardano#how-to-set-up-the-iohk-binary-caches) to avoid unnecesary builds.** If you are using Mac and you run into a segmentation fault running nix-shell, you can try running it as `GC_DONT_GC=1 nix-shell` as this [issue suggest](https://github.com/NixOS/nix/issues/4246)._
```